my_publisher_class:
  param_int: 5
  param_float : 2.7
  param_string : "sensor motion robot engineering is not difficult for me"